config/locales/en/summary.yml

```diff
@@ -535,15 +535,19 @@ en:
 
       # BEGIN other outgoings details section
       income_tax_rate_above_threshold:
-        question: In the last 2 years, has your client paid the 40% income tax rate?
+        question: Client paid 40% Income Tax in last 2 years?
+        answers:
+          <<: *YESNO
+      partner_income_tax_rate_above_threshold:
+        question: Partner paid 40% Income Tax in last 2 years?
         answers:
           <<: *YESNO
       outgoings_more_than_income:
-        question: Are your client’s outgoings more than their income?
+        question: Outgoings more than income?
         answers:
           <<: *YESNO
       how_manage:
-        question: How does your client manage if their outgoings are more than their income?
+        question: How they cover outgoings that are more than their income
```

Comment:
Could the text have remained the same as before just with `does your client` changed to `do they`

Reply:
This is from the figma - so I just did as I was told! 😂

Reply:
Ah my bad haha